generators/app/templates/src/utils/array.ts

```diff
@@ -8,12 +8,15 @@ import Immutable from 'seamless-immutable';
  * @param {string} namespace prefix for the resulting values
  * @returns {object} (['A', 'B', 'C', 'name']) -> { A: '@name/A', B: '@name/B', C: '@name/C' }
  */
-export function stringArrayToObject(actionsArray, namespace = '') {
-  if (actionsArray.some(actionName => !actionName || typeof actionName !== 'string')) {
+export function stringArrayToObject(actionsArray: string[], namespace = ''): { [key: string]: string } {
+  if (actionsArray.some((actionName: string) => !actionName || typeof actionName !== 'string')) {
     throw new Error('Action names must be strings and must not be empty');
   }
 
-  return new Immutable(actionsArray).asObject(actionName => [actionName, `${namespace}:${actionName}`]);
+  return new (Immutable as any)(actionsArray).asObject((actionName: string) => [
```

Comment:
what's this Immutable as any? Why do you need to do this?

Reply:
Had forgotten about this any but it was because TS was demanding the type for immutable and I couldn't find it. Will look for it though, don't worry